src/script/api/script_list.cpp

```diff
@@ -403,6 +403,44 @@ class ScriptListSorterItemDescending : public ScriptListSorter {
 
 
 
+bool ScriptList::SaveObject(HSQUIRRELVM vm)
+{
+	sq_pushstring(vm, "List");
+	sq_newtable(vm);
+	sq_pushinteger(vm, this->sorter_type);
+	sq_pushbool(vm, this->sort_ascending);
+	sq_rawset(vm, -3);
+	return true;
```

Comment:
Is this `return` wrong, or shouldn't the code after this be here?

Reply:
Definitely a leftover from testing.